### DISCUSSION TITLE
Get LTSP users from different machines

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#798] [General]
I did put an iPXE line for booting an LTSP image using users from server (it works fine), (Mounted with sshfs). it's normal boot. 

I'll like to be able to get users from a different UBUNTU machine instead of ltsp server. (Mounted with sshfs) 
Can be achieved using LTSP.CONF, iPxe parámeter or any other way?

**Comment by alkisg** [comment]
You can use the `SSH_SERVER` ltsp.conf variable, but you'd also need to include its SSH keys in ltsp.img.

**Comment by Soylor** [comment]
I did not find SSH_SERVER documented in ltsp.conf manpage. Should i put it on server, common or clientes section?
It's SSH_SERVER=ip or SERVER_SERVER="IP" ?

I know that SSH keys are copied with ltsp image or ltsp initrd (one of them)
Where are they copied to?

**Reply by Soylor** [reply]
Any help?

**Comment by Soylor** [comment]
So i should:

put Ssh_Server=ip in server section?
Copy ssh public key to somewhere inside chroot (where?)

**Comment by alkisg** [comment]
Sorry, no time for detailed instructions, you may read the source code there:

https://github.com/ltsp/ltsp/blob/main/ltsp/common/initrd/35-initrd.sh#L40

**Comment by Soylor** [comment]
It’s perfect. Thank you.

SSH_SERVER=IP is correct? Is it placed in server section?
